src/main/java/com/checkmarx/flow/CxFlowRunner.java

```diff
@@ -484,11 +484,13 @@ private ScanResults scan(ScanRequest request) throws ExitThrowable {
     }
 
     private boolean isScaEnabled() {
```

Comment:
each VulnerabilityScanner has a isEnabled function.
having this function in CxFlowRunner is a logical bug!